src/tactic/ui/panel/tile_layout_wdg.py

```diff
@@ -1633,7 +1632,13 @@ def get_scale_wdg(my):
 spt.tile_layout.image_drag_action = function(evt, bvr, mouse_411) {
 
     var dst_el = spt.get_event_target(evt);
-    var dst_top = dst_el.getParent(".spt_tile_top");
```

Comment:
a better way to write this is
var dst_top = dst_el.hasClass("spt_tile_top") ? dst_el : dst_el.getParent(".spt_tile_top");